qnx_qemu/build/BUILD

```diff
@@ -49,8 +49,13 @@ qnx_ifs(
         "system.build",
         ":scripts",
         ":configs",
+        "@scrample//src:scrample",
```

Comment:
I'm guessing we need a separate target for a non minimal image. More like a demo image?